sass/cd/cd-header/_cd-dropdowns-drupal.scss

```diff
@@ -5,34 +5,37 @@
   .expanded {
 
     // Hide child nav.
-    & > ul {
+    > ul {
       display: none;
     }
 
     // Add down arrow.
-    & > a {
+    > a {
       position: relative;
       &:after {
-        margin-left: 2px;
-        color: #EB5C6D;
-        content: '\e958';
-        display: inline-block;
-        font-family: 'unocha-icons';
+        position: relative;
+        content: "";
+        height: 12px;
+        width: 12px;
         margin-left: 4px;
+        background-image: url(../img/icons/arrow-down--highlight-red.svg);
+        background-repeat: no-repeat;
+        background-position: 0 50%;
+        background-size: 12px 12px;
       }
     }
 
     &.open {
 
-      & > ul {
+      > ul {
         display: block;
         z-index: map-get($zindex, dropdown);
       }
 
       // Change to up arrow.
-      & > a {
+      > a {
         &:after {
-          content: '\e957';
+          background-image: url(../img/icons/arrow-up--highlight-red.svg);
         }
       }
     }
```

Comment:
@rupl This file was my first attempt to factor in differences between styleguide main menu dropdown and Drupal. It might be a candidate for the extend technique used in user menu, but let's tackle that separately. Just want to point it out for now.

Reply:
👍